angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/option-link.html

```diff
@@ -17,6 +16,5 @@
            <option ng-repeat='obj in portlet.widgetData[config.arrayName]' value="{{obj[config.value]}}" title='{{obj[config.display].length > 26 ? obj[config.display] : ""}}'>{{obj[config.display] | trimMiddle:26}}</option>
        </select>
     </div>
-  </div>
```

Comment:
Talked f2f in 3233 because it was unclear to me too.  Looking at whole file made it clearer while talking f2f.